config-connector/solutions/README.md

```diff
@@ -30,13 +30,14 @@ The full list of solutions grouped by area:
 
 ### helm
 
-These samples are consumable as [helm charts](https://helm.sh/docs/topics/charts/).
+These solutions are consumable as [helm charts](https://helm.sh/docs/topics/charts/).
 Common targets for modification are listed in `values.yaml`.
 
 * [Installing helm 3.0](https://helm.sh/docs/intro/install/). These solutions have been verified with Helm v.3+.
-* Showing values: `helm show PATH_TO_CHART`
-* Validating chart: `helm template PATH_TO_CHART`
-* Setting chart: `helm install PATH_TO_CHART -generate-name`
+* Showing values: `helm show values PATH`
+* Validating chart: `helm template PATH`
+* Setting chart: `helm install PATH -generate-name`
+Where `PATH` is the path to the relevant solution folder.
```

Comment:
You want to add the description of the PATH to each bullet point, or add a new line between it and the list.
Currently it is in the last bullet point.